web/package.json

```diff
@@ -10,7 +10,7 @@
     "node": ">=16"
   },
   "scripts": {
-    "start": "vite",
+    "start": "vite --port 3000",
```

Comment:
Add setting to `vite.config.ts` instead of adding arguments

https://vitejs.dev/config/preview-options.html#preview-port

```ts
export default defineConfig({
  server: {
    port: 3000
  },
  // ...
});
```